packages/block-editor/src/store/selectors.js

```diff
@@ -560,18 +560,18 @@ export function getSelectedBlocksInitialCaretPosition( state ) {
 }
 
 /**
- * Returns the current multi-selection set of block client IDs, or an empty
- * array if there is no multi-selection.
+ * Returns the current selection set of block client IDs (multiselection or single selection).
  *
  * @param {Object} state Editor state.
  *
  * @return {Array} Multi-selected block client IDs.
  */
-export const getMultiSelectedBlockClientIds = createSelector(
+export const getSelectedBlockClientIds = createSelector(
```

Comment:
Could do for some separate unit tests to distinguish the two functions.

Aside: This selector function confuses me as to what its doing with the slices and root traversal below.